bin/interpreter.sh

```diff
@@ -86,6 +89,9 @@ ZEPPELIN_SERVER=org.apache.zeppelin.interpreter.remote.RemoteInterpreterServer
 INTERPRETER_ID=$(basename "${INTERPRETER_DIR}")
 ZEPPELIN_PID="${ZEPPELIN_PID_DIR}/zeppelin-interpreter-${INTERPRETER_ID}-${ZEPPELIN_IDENT_STRING}-${HOSTNAME}.pid"
 ZEPPELIN_LOGFILE="${ZEPPELIN_LOG_DIR}/zeppelin-interpreter-"
+if [[ ! -z "$INTERPRETER_NAME" ]]; then
+    ZEPPELIN_LOGFILE+="${INTERPRETER_NAME}-"
+fi
```

Comment:
`INTERPRETER_NAME` and `INTERPRETER_ID` are used together ? I assume the spark1 log file should be something like `zeppelin-interpreter-spark1-jzhang-jzhangMBPr.local.log` and spark2 log is  `zeppelin-interpreter-spark2-jzhang-jzhangMBPr.local.log`